packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -49,38 +50,75 @@ The **Custom Sign-In Page** offers basic and advanced customization options to c
-2. Use the form fields on the left side of the page to customize links, labels, and headings.
+1. Click the **Labels** tab.
+2. Locate the component type that you want to update and click **Edit**.
 
 You can also customize the placeholder text that appears in recovery flows when end users click account recovery links (for example, Forgot password and Unlock account). If you leave a label field blank, Okta uses the default text.
 
 > **Note:** Although Okta displays default labels, links, and headings in the end user's display language or browser language, Okta doesn't display localized versions of customized text and links. Text that you change here is hard-coded. To specify multiple localized versions of headings and labels, use the [Sign-in Widget text configuration options](https://github.com/okta/okta-signin-widget/#language-and-text).
 
 ### Use the embedded HTML editor
 
-If you are familiar with using HTML and want to change the page layout, colors, button shapes, and other elements, use the embedded HTML editor in the middle of the page. You can modify the current HTML/CSS and JavaScript or paste your own code.
+Use the code editor to modify any HTML, CSS, or JavaScript on the widget. See [Customization examples](#customization-examples) for snippets that you can update and use.
+
+> **Note:** There is no draft mode in the code editor. When you click **Publish**, your changes go live. If you want to test changes without impacting production, use a test or development org.
+
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Sign-in page** box, click **Edit**.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+
+   > **Note:** The code editor toggle only appears if you connect to Okta using a [custom domain](/docs/guides/custom-url-domain/) or use the [Brands API](#use-the-brands-api).
+
+4. Make changes directly in the editor. If you enter `{`, `(`, or `.` you see a list of available variables that you can use. See [Use variables](#use-variables).
+   <!-- * Click **Preview** to see your changes in a new browser window before you publish. -->
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
 
-1. Make changes directly in the embedded editor.
-2. Click **Preview** to preview your changes before you publish.
-3. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
-4. Click **Save and Publish** when you finish.
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. The console restores the default HTML/CSS and JavaScript code.
```

Comment:
we got rid of auto-save

Reply:
Updated in custom error pages guide, too.